src/main/resources/application-worker.yaml

```
@@ -17,7 +17,7 @@ rhsm-subscriptions:
   hourlyTallyEventBatchSize: ${HOURLY_TALLY_EVENT_BATCH_SIZE:16000}
   inventory-service:
     datasource:
-      url: jdbc:postgresql://${INVENTORY_DATABASE_HOST:localhost}/${INVENTORY_DATABASE_DATABASE:insights}?ApplicationName=${clowder.metadata.name:rhsm-subscriptions}
+      url: jdbc:postgresql://${INVENTORY_DATABASE_HOST:localhost}/${INVENTORY_DATABASE_DATABASE:insights}?currentSchema${INVENTORY_DATABASE_SCHEMA:public}&ApplicationName=${clowder.metadata.name:rhsm-subscriptions}
```

I think it's missing the "=":
```suggestion
      url: jdbc:postgresql://${INVENTORY_DATABASE_HOST:localhost}/${INVENTORY_DATABASE_DATABASE:insights}?currentSchema=${INVENTORY_DATABASE_SCHEMA:public}&ApplicationName=${clowder.metadata.name:rhsm-subscriptions}
```

---

yes, thanks!